src/helm/proxy/clients/http_client.py

```diff
@@ -0,0 +1,151 @@
+from dataclasses import asdict
+
+from helm.common.cache import Cache, CacheConfig
+from helm.common.request import (
+    Request,
+    RequestResult,
+    Sequence,
+    Token,
+    EMBEDDING_UNAVAILABLE_REQUEST_RESULT,
+)
+from helm.common.tokenization_request import (
+    DecodeRequest,
+    DecodeRequestResult,
+    TokenizationRequest,
+    TokenizationRequestResult,
+    TokenizationToken,
+)
+from .client import Client, wrap_request_time
+
+import requests
+
+
+class HTTPClient(Client):
```

Comment:
Name suggestions:

- If we want NeurIPS efficiency challenge branding, then: `NeuripsEfficiencyChallengeClient`, `EfficiencyChallengeClient`
- Otherwise: `HTTPInferenceClient`, `HTTPModelClient`

Reply:
The branding doesn't matter too much for us